src/Files.App/Views/Layouts/ColumnsLayoutPage.xaml.cs

```diff
@@ -96,7 +96,8 @@ protected override void OnNavigatedTo(NavigationEventArgs eventArgs)
 
 			if (!string.IsNullOrEmpty(pathRoot))
 			{
-				var rootPathList = App.QuickAccessManager.Model.PinnedFolders.Select(NormalizePath)
+				var service = Ioc.Default.GetRequiredService<IQuickAccessService>();
```

Comment:
Shouldn't the service be defined at the top of the class?

Reply:
You're right, I was too lazy to 😅